third_party/3/lxml/etree.pyi

```diff
@@ -23,7 +23,12 @@ class ElementChildIterator(Iterator['_Element']):
 
 class _Element(Iterable['_Element']):
     def addprevious(self, element: '_Element') -> None: ...
-
+    def addnext(self, element: '_Element') -> None: ...
+    def clear(self) -> None: ...
+    def get(self, key: _AnyStr, default: Optional[_AnyStr] = None) -> _AnyStr: ...
+    def xpath(self, _path: _AnyStr, namespaces: Optional[_DictAnyStr] = None, extensions: Any = None, smart_strings: bool = True, **_variables) -> Any: ...
```

Comment:
Can you change all the defaults (here and in other functions) to `...` instead of `None`? Also, please add a type (probably `Any`) for `_variables`.

Reply:
I thought that ... was semantically equivalent to `pass`. So I went to read a bit on it . My understanding now is that it resolves to Ellipsis, an instance of the ellipsis class, and that it can be used also for default values.
Is the rationale for it to avoid an unneeded duplication and risk of conflict with the actual implementation ? In that case, this should affect all the default values, not only the None ones, right ?

Reply:
Yes, all defaults should be `...`, but I think all defaults here are `None`. We do this to avoid adding unnecessary information in typeshed, because that introduces maintenance overhead.